cypress-tests/cypress/e2e/ConnectorUtils/Adyen.js

```diff
@@ -39,6 +43,8 @@ export const connectorDetails = {
         "paymentSyncStatus": "processing",
         "refundStatus": "pending",
         "refundSyncStatus": "pending",
+        "customer_acceptance":null,
+        "setup_future_usage": "on_session",
```

Comment:
as discussed, please remove this as this field is not used anywhere else and this will create confusion